palanaeum/templates/palanaeum/elements/entry_li.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
-<article class="entry-article w3-display-container w3-border w3-card {% if not entry.is_visible %}hidden-element{% endif %} {% if entry.is_suggestion %}entry-suggestion{% endif %}" id="e{{ entry.id }}" data-entry-id="{{ entry.id }}" data-order="{{ entry.order }}">
+<a name="e{{ entry.id }}" class="entry-anchor"></a>
+<article class="entry-article w3-display-container w3-border w3-card {% if not entry.is_visible %}hidden-element{% endif %} {% if entry.is_suggestion %}entry-suggestion{% endif %}" data-entry-id="{{ entry.id }}" data-order="{{ entry.order }}">
```

Comment:
We need to have IDs on the <article> elements, many other scripts assume that there is an article like "#e324" and assume there's content inside. I see you deleted the "id" attribute. A simple copy function in copy_and_share.js will not work as it is right now.

Reply:
That's good to know. The problem with having the ID element match the name of the anchor is that the ID overrules the anchor. So scrolling would still choose the main entry element as target and hence have the header overlap.

If the ID is required in the frontend code, I will use `#entry324` rather than `#e234`. This way, we won't change the behaviour of existing links while allowing the JS to still work.

Reply:
Yeah, that sounds reasonable :)